DfE.FindInformationAcademiesTrusts/Pages/Trusts/Contacts/ContactsAreaModel.cs

```diff
@@ -5,30 +5,36 @@
 using DfE.FindInformationAcademiesTrusts.Services.Trust;
 using Microsoft.AspNetCore.Mvc;
 
-namespace DfE.FindInformationAcademiesTrusts.Pages.Trusts;
+namespace DfE.FindInformationAcademiesTrusts.Pages.Trusts.Contacts;
 
-public class ContactsModel(
+public class ContactAreaModel(
     IDataSourceService dataSourceService,
-    ILogger<ContactsModel> logger,
-    ITrustService trustService)
-    : TrustsAreaModel(dataSourceService, trustService, logger, "Contacts")
+    ITrustService trustService,
+    ILogger<ContactAreaModel> logger,
+    Func<TrustSubNavigationLinkModel, bool> setActivePage
+)
+    : TrustsAreaModel(dataSourceService, trustService, logger, ViewConstants.ContactsPageTitle)
 {
     public Person? ChairOfTrustees { get; set; }
     public Person? AccountingOfficer { get; set; }
     public Person? ChiefFinancialOfficer { get; set; }
     public InternalContact? SfsoLead { get; set; }
     public InternalContact? TrustRelationshipManager { get; set; }
 
-    public const string ContactNameNotAvailableMessage = "No contact name available";
```

Comment:
I think these consts are better either here or directly in `_DisplayContact.cshtml` than in `ViewConstants` because they are only used in one contacts-related place (whereas `ViewConstants` is more for site-wide things)